terraform/deployments/search-api-v2/discovery_engine.tf

```diff
@@ -86,7 +85,6 @@ resource "aws_secretsmanager_secret_version" "discovery_engine_configuration_sea
   secret_string = jsonencode({
     "GOOGLE_CLOUD_CREDENTIALS"        = base64decode(google_service_account_key.search_admin.private_key)
     "DISCOVERY_ENGINE_ENGINE"         = google_discovery_engine_search_engine.govuk.name
-    "DISCOVERY_ENGINE_SERVING_CONFIG" = local.discovery_engine_site_search_serving_config_path
```

Comment:
We no longer need this - Search Admin can figure out serving config names from the engine name defined above.